packages/pigeon/lib/swift_generator.dart

```diff
@@ -214,7 +214,7 @@ class SwiftGenerator extends StructuredGenerator<SwiftOptions> {
           final String fieldName = field.name;
           final String nullsafe = field.type.isNullable ? '?' : '';
           if (field.type.isClass) {
```

Comment:
This is now identical to the final `else`, just written slightly differently; they should be consolidated and normalized (either with everything using `fieldName`, or with `fieldName` eliminated in favor of `field.name`).